<launch>
    <arg name="queue_size" default="5" />
    <arg name="camera_namespace" default="/head_xtion" />
    <arg name="upper_body_detections" default="/upper_body_detector/detections" />
    <arg name="autostart" default="true"/>

    <!-- 1814400 = 3*7*24*60*60 = 3 full weeks. -->
    <arg name="expected_runtime" default="1814400" />

    <include file="$(find ground_plane_estimation)/launch/ground_plane_fixed.launch"/>
    <include file="$(find upper_body_detector)/launch/upper_body_detector.launch"/>

    <node pkg="strands_head_orientation" type="store_detections" name="store_detections" output="screen">
        <param name="queue_size" value="$(arg queue_size)" type="int"/>
        <param name="camera_namespace" value="$(arg camera_namespace)" type="string"/>
        <param name="upper_body_detections" value="$(arg upper_body_detections)" type="string"/>
        <param name="autostart" value="$(arg autostart)" type="bool"/>
        <param name="expected_runtime" value="$(arg expected_runtime)" type="int"/>
    </node>
</launch>
